integration-tests/README.md

```diff
@@ -2,6 +2,35 @@
 
 This directory contains integration tests that make actual API calls to Dynatrace services. These tests are designed to verify the functionality of the MCP server against a real Dynatrace environment.
 
+## Test Suites
```

Comment:
we don't need additional information about the test-suites in here. Please remove these again.